templates/blog/post_detail.html

```diff
@@ -0,0 +1,25 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+{% block content %}
+  <p>{{ post.owner }}</p>
+  <p>title: {{ post.title }}</p>
+  <p>created at: {{ post.created_time}}</p>
+  <h6>{{ post.content }}</h6>
+  <br><br>
+  <p>Comments:</p>
+
+  {% for commentary in post.commentaries.all %}

```

Comment:
The syntax `post.commentaries.all` should be `post.commentaries.all()` to correctly call the `all()` method and retrieve all related commentaries.